.github/workflows/docs.yaml

```diff
@@ -22,12 +23,11 @@ jobs:
         env:
           BUILD_DIR: docs
           BUILD_ONLY: true
-          TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: deploy
+      - name: Deploy
         if: github.ref == 'refs/heads/main'
-        uses: shalzz/zola-deploy-action@v0.14.1
-        env:
-          BUILD_DIR: docs
-          PAGES_BRANCH: gh-pages
-          TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: peaceiris/actions-gh-pages@v3
```

Comment:
I could be wrong but doesnt there need to be a build step in here? I think before this was handled by `zola-deploy-action` but is that the case with this also? With Hugo it looks like there needs to be setup and build before publishing:
```
      - name: Setup Hugo
        uses: peaceiris/actions-hugo@v2
        with:
          hugo-version: '0.85.0'

      - name: Build
        run: hugo --minify
```

Reply:
The previous step does the build. The new one picks up the content of the `docs/public` folder and publishes it